tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,153 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Basic aistack test
+
+# Summary: This test performs the following actions
+#  - Create a VM in EC2 using SLE-Micro-BYOS later than 6.0 version
+#  - Install the required dependencies to install the aistack helm chart
+#  - Test access to OpenWebUI and run integration tests with Ollama and MilvusDB
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+#
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use transactional;
+use containers::k8s;
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use transactional qw(process_reboot trup_install trup_shell);
+use File::Basename;
+use version_utils;
+use Data::Dumper;
+
+sub install_dependency_package {
+    my ($instance, $rke2_url, $kubectl_url, $helm_url) = @_;
+    my $runtime = get_required_var('PUBLIC_CLOUD_AISTACK_DEP_PKG');
+
+    for (split(',\s*', $runtime)) {
+        my $package = $_;
+        my $pkg_status = script_run("rpm -q $package");
+        my $tr_install_cmd = "";
+        record_info("Package status $pkg_status");
+
+        if ($pkg_status == 0) {
+            record_info("$package is already installed");
+        } else {
+            record_info("$package is not installed, installing now...");
+            if ($package eq "rke2-server") {
+                script_run("sudo transactional-update shell", timeout => 100);
+                script_run("curl -sSL $rke2_url | sudo sh", timeout => 100);
+                script_run("exit", timeout => 100);
```

Comment:
```suggestion
                $tr_install_cmd .= "curl -sSL $rke2_url | sudo sh";
```
Here too, it makes sense the string preparation for next trup_shell execution, even simpler. 
Unless you found any `wait_serial` execution issue: did you ?

Reply:
`     record_info("$package is not installed, installing now...");
            if ($package eq "rke2-server") {
                $tr_install_cmd .= "curl -sSL $rke2_url | sudo sh; ";
                trup_shell("$tr_install_cmd", reboot => 1, timeout => 200);
            } elsif ($package eq "helm") {
                $tr_install_cmd .= "[ ! -d /usr/local/bin ] && mkdir -p /usr/local/bin; ";
                $tr_install_cmd .= "sudo chmod 755 /usr/local/bin; ";
                $tr_install_cmd .= "curl -sSL $helm_url | sudo sh; ";
                $tr_install_cmd .= "echo 'export PATH=\$PATH:/usr/local/bin' >> ~/.bashrc; ";
                $tr_install_cmd .= "source ~/.bashrc; ";
                $tr_install_cmd .= "helm version; ";
                trup_shell("$tr_install_cmd", reboot => 1, timeout => 200);
            } elsif ($package eq "kubectl") {
                $tr_install_cmd .= "curl -LO $kubectl_url; ";
                $tr_install_cmd .= "chmod +x ./kubectl; ";
                $tr_install_cmd .= "mkdir -p ~/bin; ";
                $tr_install_cmd .= "mv ./kubectl ~/bin/kubectl; ";
                $tr_install_cmd .= "echo 'export PATH=\$PATH:\$HOME/bin' >> ~/.bashrc; ";
                $tr_install_cmd .= "source ~/.bashrc; ";
                $tr_install_cmd .= "kubectl version --client; ";
                trup_shell("$tr_install_cmd", reboot => 1, timeout => 200);
            }
            if (($package eq "docker") || ($package eq "curl") || ($package eq "git")) {
                trup_call("pkg install $package");
                record_info("Installed $package.");
            }
        }    # pkg_status if
             # Execute all commands with trup_shell and reboot
        if ($tr_install_cmd) {
            record_info("Executed command $tr_install_cmd");
            record_info("Installed $package.");
        } else {
            process_reboot(trigger => 1);
            record_info("Reboot Completed");
        }
`